### ISSUE TITLE
jackson core dependency missing with a specific setup

### ISSUE THREAD

**Issue by mtsbarbosa** [jgrodziski/keycloak-clojure#43]
When having the following structure:

project.clj
```clojure
:dependencies [[org.clojure/clojure "1.11.1"]
                           [io.pedestal/pedestal.service "0.5.10"]

                           [io.pedestal/pedestal.jetty "0.5.10"]
                           [keycloak-clojure "1.27.0"]
                           [org.clojure/data.json "0.2.6"]
                           [org.clojure/java.jdbc "0.7.12"]
                           [mysql/mysql-connector-java "8.0.29"]
                           [com.outpace/config "0.13.5"]
                           [org.slf4j/slf4j-simple "1.7.28"]]
```

and following keycloak-clojure usage:

ports.keycloak.core.clj
```clojure
(def keycloak-deployment (kc-deploy/deployment (kc-deploy/client-conf "http://localhost:8090"
                                                                      "myrealm"
                                                                      "my-client")))
```

I have a compilation error as such:
```shell
{:clojure.main/message
 "Execution error (ClassNotFoundException) at java.net.URLClassLoader/findClass (URLClassLoader.java:382).\ncom.fasterxml.jackson.core.util.JacksonFeature\n",
 :clojure.main/triage
 {:clojure.error/class java.lang.ClassNotFoundException,
  :clojure.error/line 382,
  :clojure.error/cause
  "com.fasterxml.jackson.core.util.JacksonFeature",
  :clojure.error/symbol java.net.URLClassLoader/findClass,
  :clojure.error/source "URLClassLoader.java",
  :clojure.error/phase :execution},
 :clojure.main/trace
 {:via
  [{:type clojure.lang.Compiler$CompilerException,
    :message "Syntax error macroexpanding at (core.clj:16:26).",
    :data
    {:clojure.error/phase :execution,
     :clojure.error/line 16,
     :clojure.error/column 26,
     :clojure.error/source "core.clj"},
    :at [clojure.lang.Compiler$InvokeExpr eval "Compiler.java" 3719]}
   {:type clojure.lang.ExceptionInfo,
    :message "Failed to build the keycloak app client",
    :data
    {:credentials {:secret nil},
     :verify-token-audience true,
     :auth-server-url "http://localhost:8090",
     :policy-enforcer {},
     :realm "bakery-keycloak",
     :use-resource-role-mappings true,
     :resource "my-backend",
     :ssl-required "external",
     :confidential-port 0},
    :at
    [keycloak.deployment$deployment invokeStatic "deployment.clj" 23]}
   {:type java.lang.NoClassDefFoundError,
    :message "com/fasterxml/jackson/core/util/JacksonFeature",
    :at
    [com.fasterxml.jackson.databind.ObjectMapper
     <init>
     "ObjectMapper.java"
     655]}
   {:type java.lang.ClassNotFoundException,
    :message "com.fasterxml.jackson.core.util.JacksonFeature",
    :at
    [java.net.URLClassLoader findClass "URLClassLoader.java" 382]}],
  :trace
  [[java.net.URLClassLoader findClass "URLClassLoader.java" 382]
```

I have found out a workaround, to add the jackson-core to deps:
```clojure
[com.fasterxml.jackson.core/jackson-core "2.13.2"]
```

**Comment by jgrodziski** [comment]
Should be fixed starting from `1.28.0`, could you test it?

**Comment by mtsbarbosa** [comment]
@jgrodziski it fixed the issue for me
thank you!
